client/js/keybinds.js

```diff
@@ -78,6 +78,33 @@ Mousetrap.bind(["alt+shift+up", "alt+shift+down"], function (e, keys) {
 	return false;
 });
 
+// Switch to the next/previous unread chat
+Mousetrap.bind(["alt+mod+up", "alt+mod+down"], function (e, keys) {
```

Comment:
@Nachtalb can we test this? If you extract it to a new function it should be easy to call and mock jumpToChannel to make sure the right channel is passed.

Reply:
Quite some time since I worked on this. What did you want me to do exactly, put it in its own function so we can write a test for it? If so, where should I put the function?

Reply:
> put it in its own function so we can write a test for it?

Guess that'd be the idea yeah...
As for where to put it... we could make it a method of the client, rather than open coding it in the keybinds.
Can't put it at the network level as we want to jump between them, so that'd be the next one up.

Reply:
no worries. Haven't touched this project in years. So I can't remember all the stuff either. XD

Reply:
If I were to put it in its own function rather than as it is now. We should also do this for the rest of the keybinds. This should be done in its own PR thought. To keep everything consistent. Doing this in this PR. will make it grand instead of as small as PR should be.

Reply:
IIRC our client tests are also broken since the TS rewrite... so it'll be some effort we shouldn't punt on this